packages/rss-handler/src/index.ts

```diff
@@ -214,13 +243,34 @@ const sendUpdateSubscriptionMutation = async (
   }
 }
 
+const isItemRecentlySaved = async (
+  redisClient: RedisClient,
+  userId: string,
+  url: string
+) => {
+  const key = `recent-saved-item:${userId}:${url}`
+  const result = await redisClient.get(key)
```

Comment:
Same thing about exception, if it fails to connect to redis i think we can just continue and assume the item isn't saved already

Reply:
noted